src/core/pages/__mocks__/_user_guide.md

```
@@ -0,0 +1,174 @@
+
+[Multi-Criteria Decision Analysis](#multi-criteria-decision-analysis)
+
+[Area selection](#area-selection)
+
+[AI insights](#ai-insights)
+
+## Multi-Criteria Decision Analysis
+
+![](https://www.youtube.com/embed/g7WMD10DMPs?si=Gl6RdNM0L3ufi0uF##800,470,1)
+
+<iframe width="800" height="470" src="https://www.youtube.com/embed/g7WMD10DMPs?si=Gl6RdNM0L3ufi0uF" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin"></iframe>
```

Markdownlint

Element: iframe
Inline HTML

(MD033, no-inline-html)